### ISSUE TITLE
Tries to identify un-identifiable items (gems)

### ISSUE THREAD

**Issue by Kellen-Stuart** [aeon0/botty#594]
**Botty Version**
What version of botty are you running? Are you using a pre-compiled release? Which one. Are you using the latest master branch? On which commit?

Botty 0.6.7-dev

**Describe the bug**
Botty will attempt to identify flawless gems. This _may_ be a feature request, but botty needs to retain a boolean which states that the items it has picked up are identifiable. Perhaps he keeps a memory of what he has picked up since the last identification. 

I believe this would improve the performance of botty for people who are picking up non identifiable items. 

**To Reproduce**

1. Pickup all flawless gems => pickit.ini 
2. Run botty
3. Watch botty try to identify flawless gem

**Expected behavior**
Botty should not try to identify items that you can't identify.

**Screenshots / Video**
Sorry I do not have a video, but I can get one if you need.

**Logs**
Don't have the exact log, but there is a log message that states "THIS SHOULD NEVER HAPPEN" when he tries to identify. 

Good news is it doesn't crash the bot. Just wastes time. 

**Additional info**
I may pick this up as my first contribution to botty. :)

**Comment by mgleed** [comment]
This is in the pipeline. Will be a part of #588 . Thanks for filling out the template. :)

**Comment by Kellen-Stuart** [comment]
Thanks! I wanted to also add some debug logs. I caught botty in the act.

```
[2022-02-27 12:14:19,077] INFO       Picking up: misc_flawless_topaz (97.4% confidence)
[2022-02-27 12:14:23,271] INFO       Picking up: misc_gold (95.4% confidence)
[2022-02-27 12:14:26,772] INFO       End game. Elapsed time: 81.77s
[2022-02-27 12:14:27,040] ERROR      Error sending Discord embed: 'DiscordEmbeds' object has no attribute '_webhook'
[2022-02-27 12:14:27,041] INFO       Starting game #21
[2022-02-27 12:14:27,041] DEBUG      Waiting for templates: ['MAIN_MENU_TOP_LEFT', 'MAIN_MENU_TOP_LEFT_DARK', 'A5_TOWN_0', 'A5_TOWN_1', 'A4_TOWN_4', 'A4_TOWN_5', 'A3_TOWN_0', 'A3_TOWN_1', 'A2_TOWN_0', 'A2_TOWN_1', 'A2_TOWN_10', 'A1_TOWN_1', 'A1_TOWN_3']
[2022-02-27 12:14:29,411] DEBUG      Found Match: MAIN_MENU_TOP_LEFT (100.0% confidence)
[1039, 47, 203, 60] (1143, 85)
[2022-02-27 12:14:30,538] DEBUG      Saved character template found and already highlighted, continue
[2022-02-27 12:14:30,538] DEBUG      Waiting for templates: ['MAIN_MENU_TOP_LEFT', 'MAIN_MENU_TOP_LEFT_DARK']
[2022-02-27 12:14:30,554] DEBUG      Found Match: MAIN_MENU_TOP_LEFT (100.0% confidence)
[2022-02-27 12:14:30,554] DEBUG      Wait for Play button
[2022-02-27 12:14:30,568] DEBUG      Found Play Btn
[2022-02-27 12:14:31,443] DEBUG      Waiting for templates: ['A5_TOWN_0', 'A5_TOWN_1', 'A4_TOWN_4', 'A4_TOWN_5', 'A3_TOWN_0', 'A3_TOWN_1', 'A2_TOWN_0', 'A2_TOWN_1', 'A2_TOWN_10', 'A1_TOWN_1', 'A1_TOWN_3']
[2022-02-27 12:14:38,481] DEBUG      Found Match: A5_TOWN_1 (79.9% confidence)
[2022-02-27 12:14:38,708] INFO       Health Manager is now pausing
[2022-02-27 12:14:38,741] INFO       Capabilities: CharacterCapabilities(can_teleport_natively=True, can_teleport_with_charges=False)
[2022-02-27 12:14:39,466] DEBUG      Will pickup: {'rejuv': 4, 'health': 0, 'mana': 0}
[2022-02-27 12:14:39,478] INFO       Identifying items
[2022-02-27 12:14:39,478] DEBUG      Traverse from a5_town_start to a5_qual_kehk
[2022-02-27 12:14:49,011] ERROR      Could not find identify btn. Should not happen! Continue...
[2022-02-27 12:14:51,179] INFO       Stashing items
```

**Comment by mgleed** [comment]
Solved in #588
